code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -140,6 +140,7 @@
 	if (T && AIStatus == AI_Z_OFF)
 		SSidlenpcpool.idle_mobs_by_zlevel[T.z] -= src
 
+	QDEL_LIST(actions)
```

Comment:
More than just simple mobs use the actions list. We've typically handled that on the mob itself, where unique actions are handled---or on the action itself for specific special use cases.